tests/device_selection/test_predictor_ml.py

```
@@ -49,26 +54,80 @@ def test_setup_device_predictor_with_prediction(path_uncompiled_circuits: Path,
+    success = setup_device_predictor(
+        devices=device,
+        figure_of_merit="expected_fidelity",
+        path_uncompiled_circuits=path_uncompiled_circuits,
+        path_compiled_circuits=path_compiled_circuits,
+        gnn=True,
+        verbose=False,
+    )
+    assert success
+
+    data_path = get_path_training_data() / "training_data_aggregated"
+    if gnn:
```

Ruff (0.14.10)

Undefined name `gnn`

(F821)